Content.Server/GameObjects/Components/Disposal/DisposalMailingUnitComponent.cs

```diff
@@ -95,10 +95,10 @@ public class DisposalMailingUnitComponent : SharedDisposalMailingUnitComponent,
         private readonly List<string> _targetList = new();
 
         [ViewVariables]
-        private string _target = "";
+        private string? _target = string.Empty;
 
         [ViewVariables(VVAccess.ReadWrite)]
-        private string _tag = "";
+        private string _tag = string.Empty;
```

Comment:
Looks like there's some internal code that only checks null so these should be null.

Reply:
I think only `_target` is checked for null, not `_tag`. So I will keep `_target` nullable but without default value, and keep `_tag` like this.